src/pe_reports/report_generator.py

```diff
@@ -353,8 +359,8 @@ def generate_reports(db, datestring, data_directory, output_directory):
             file = filenames[-1]
 
             # Create folders in output directory
-            subprocess.call(["mkdir", f"{output_directory}/ppt"])
-            subprocess.call(["mkdir", f"{output_directory}/{_id}"])
```

Comment:
try:
                os.mkdir(f"{output_directory}/_id")
            except FileExistsError as err:
                logging.error(f"The output directory exists {err}", exc_info=True)
                return 0